src/Sylius/Component/Media/Model/Image.php

```diff
@@ -64,55 +63,35 @@ public function getId()
     }
 
     /**
-     * {@inheritdoc}
+     * @return string
      */
-    public function hasFile()
+    public function getMediaId()
     {
-        return null !== $this->file;
+        return $this->mediaId;
     }
 
     /**
-     * {@inheritdoc}
+     * @param string $mediaId
      */
-    public function getFile()
+    public function setMediaId($mediaId)
     {
-        return $this->file;
+        $this->mediaId = $mediaId;
     }
 
     /**
-     * {@inheritdoc}
+     * @return CmfImage
      */
-    public function setFile(\SplFileInfo $file)
+    public function getMedia()
     {
-        $this->file = $file;
-
-        return $this;
+        return $this->media;
     }
 
     /**
-     * {@inheritdoc}
+     * @param CmfImage $media
```

Comment:
As the methods declarations comes from implemented interface, `{@inheritdoc}` should be left as is.